src/Utils/Logging.hs

```diff
@@ -472,7 +474,8 @@ withElasticsearchBackend
     -> (Backend a -> IO b)
     -> IO b
 withElasticsearchBackend mgr esServer ixName inner = do
-    void $ HTTP.httpLbs putIndex mgr
+    i <- curIxName
+    createIndex i
         -- FIXME Do we need failure handling? If this fails the exeption
         -- is propagated to the main applcation
```

Comment:
Can this `FIXME` be removed? I see that `createIndex` below does have some error handling.

Reply:
The error handling in in `createIndex` doesn't address this FIXME. I think we can remove it anyways.